public/lib/events.js

```diff
@@ -34,7 +34,7 @@ var events = {
 
       // if this was a published event, do an immediate trigger
       var cache = self._cache;
-      if (cache[name]) {
+      if (cache && cache[name]) {
```

Comment:
I think this is happening in the case where an object has been destroyed or is being destroyed?